modules/edit-modes/package.json

```diff
@@ -57,6 +57,7 @@
     "@turf/difference": ">=4.0.0",
     "@turf/distance": ">=4.0.0",
     "@turf/ellipse": ">=4.0.0",
+    "@turf/envelope": "^5.1.5",
```

Comment:
is this the same as bbox?

Reply:
This is very similar to bbox, except turf envelope returns a bounding box feature instead of an array of 4 values. Do you want me to use bbox instead?

Reply:
Yes please if it is not a lot of work.

Reply:
I updated my PR - I removed the `@turf/envelope` dependency and used bbox instead.